src/libstd/sys/unix/c.rs

```diff
@@ -336,3 +345,41 @@ mod signal {
         pub sa_mask: sigset_t,
     }
 }
+
+#[cfg(target_os = "bitrig")]
+mod signal {
```

Comment:
Oh, I note you diverge from openbsd because the openbsd is wrong for `struct siginfo`.
Could you include target_os="openbsd" here (and remove of it from previous block ?)

If not possible, not a problem, I will correct it after your PR will merged.